crates/uv/src/commands/pip/tree.rs

```diff
@@ -187,19 +198,20 @@ impl<'a> DisplayDependencyGraph<'a> {
         }
 
         let mut lines = vec![line];
-
+        let empty_vec = Vec::new();
         path.push(package_name.clone());
         visited.insert(package_name.clone());
-        let required_packages = required_with_no_extra(installed_dist, self.markers);
+        let required_packages = self
+            .requires_map
+            .get(installed_dist.name())
+            .unwrap_or(&empty_vec)
```

Comment:
You can use `unwrap_or_default` here.

Reply:
seems like this gives 
```
error[E0277]: the trait bound `&Vec<PackageName>: std::default::Default` is not satisfied
    --> crates/uv/src/commands/pip/tree.rs:206:14
     |
206  |             .unwrap_or_default()
     |              ^^^^^^^^^^^^^^^^^ the trait `std::default::Default` is not implemented for `&Vec<PackageName>`
```
maybe i'm using it wrong?

Reply:
Ah, sorry. You could use `.iter().flatten()` instead, but this is fine.